src/main/resources/com/cloudbees/jenkins/plugins/bitbucket/BitbucketBuildStatusNotificationsTrait/config.jelly

```diff
@@ -3,4 +3,7 @@
   <f:entry title="${%Communicate Unstable builds to Bitbucket as Successful}" field="sendSuccessNotificationForUnstableBuild">
     <f:checkbox/>
   </f:entry>
+  <f:entry title="${%Communicate NOT_BUILT builds to Bitbucket as Failed}" field="sendFailureNotificationForNotBuiltBuild">
```

Comment:
Thanks, I did something similar on my fork to be able to work with _disableConcurrentBuilds(abortPrevious)_ option. One minor thing: maybe I'm wrong but it seems to me more logical if we'd revert that to make it like _sendSuccessNotificationForNotBuiltBuild_. For me the default case should be **failure** for all except **success**

Reply:
>For me the default case should be failure for all except success

I agree and IMO it was a mistake to implement the default as it is today (success), but I wanted to avoid a breaking change.